### PR TITLE
Add how to use with Next.js in docs

### PR DESCRIPTION
**Pull request by JonatanSalas**
Added a section on README.md related to how to integrate this package with next.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,105 @@ const { helmet } = helmetContext;
...
+        return (
+            <html lang="es" dir="ltr">
+                <Head>
+                    <meta
+                        name="viewport"
+                        content="user-scalable=0, initial-scale=1, minimum-scale=1, width=device-width, height=device-height"
+                    />
+                    {helmetContext.helmet.meta.toComponent()}
+                    {helmetContext.helmet.link.toComponent()}
+                    {helmetContext.helmet.title.toComponent()}
```
Comment (osym) [inline README.md:113]: Btw placing `Title` here triggers a warning
```
 Warning: <title> should not be used in _document.js's <Head>. https://err.sh/next.js/no-document-title
```

Comment (JonatanSalas) [reply README.md]: Fixed!

**Comment by JonatanSalas** [comment]
@osym updated the docs to reflect the need for title removal from `_document.js` file.

File: README.md [hunk 2]
```diff
@@ -66,6 +66,104 @@ const { helmet } = helmetContext;
...
+                    <Main />
+                    <NextScript />
+                </body>
+            </html>
+        );
+    }
+}
+```
+
+2. Create an `_app.js in pages folder with the following content:
```
Comment (zachguo) [inline README.md:128]: missing ending ` after _app.js.

Comment (JonatanSalas) [reply README.md]: Done!

**Pull request closed without merging**